src/legacy/core_plugins/data/public/query/query_bar/components/query_bar_top_row.tsx

```diff
@@ -44,7 +41,7 @@ import {
   getQueryLog,
 } from '../../../../../../../plugins/data/public';
 import { useKibana, toMountPoint } from '../../../../../../../plugins/kibana_react/public';
-
+import { esKuery } from '../../../../../../../plugins/data/public';
```

Comment:
Looks like we already have an import from `plugins/data/public` above that this could be moved to